permissions/actions.js

```diff
@@ -1,22 +1,30 @@
 const diff = require('deep-diff');
 
-const CLOSE_CASE = 'closeCase';
-const REOPEN_CASE = 'reopenCase';
-const CASE_STATUS_TRANSITION = 'caseStatusTransition';
-const ADD_NOTE = 'addNote';
-const EDIT_NOTE = 'editNote';
-const ADD_REFERRAL = 'addReferral';
-const EDIT_REFERRAL = 'editReferral';
-const ADD_HOUSEHOLD = 'addHousehold';
-const EDIT_HOUSEHOLD = 'editHousehold';
-const ADD_PERPETRATOR = 'addPerpetrator';
-const EDIT_PERPETRATOR = 'editPerpetrator';
-const ADD_INCIDENT = 'addIncident';
-const EDIT_INCIDENT = 'editIncident';
-const ADD_DOCUMENT = 'addDocument';
-const EDIT_DOCUMENT = 'editDocument';
-const EDIT_CASE_SUMMARY = 'editCaseSummary';
-const VIEW_POST_SURVEY = 'viewPostSurvey';
+const actionsMaps = {
```

Comment:
This actions are now grouped by what target model they work on, because when setting up the `can` function, we need to pass in the model object on which we will `allow` the actions to be performed.